File: src/GameCritic.Domain/Entities/Game.cs
using System.Collections.Generic;

namespace GameCritic.Domain.Entities
{
    public class Game : BaseEntity
    {
        public string Title { get; set; }

        public DateTime ReleaseDate { get; set; }

        public string Summary { get; set; }

        public string? ImageName { get; set; }

        public decimal Price { get; set; }

        public double? Playtime { get; set; }

        public double? Score { get; set; }

        public byte[] RowVersion { get; set; }

        public int PublisherId { get; set; }
        public Publisher Publisher { get; set; }

        public ICollection<Review>? Reviews { get; set; }

        public ICollection<Ranking>? Rankings { get; set; }

        public ICollection<GameGenre> GameGenres { get; set; }
    }
}

File: src/GameCritic.Angular/src/app/models/game/game.ts
import { PublisherList } from "../publisher/publisher-list";
import { GenreList } from "../genre/genre-list";
import { GameAwardList } from "../ranking/game-award-list";

export interface Game {
    id: number,
    title: string,
    releaseDate: Date,
    summary: string,
    imageName: string,
    price: number,
    playtime: number,
    score: number,
    publisher: PublisherList,
    awards: GameAwardList[],
    genres: GenreList[]
}